ref/app-architecture.md

```diff
@@ -1,32 +1,39 @@
+# App Architecture
 
```

Comment:
It would be valuable to add a short sentence of paragraph about why you're using the Entity/Hydrator pattern, and what that is, for your team mates. Up to you; I don't require it to approve this pull request.